src/hooks/fightArenaHooks.tsx

```diff
@@ -186,6 +186,20 @@ export const useBattleLogic = (
           });
         } else {
           setBattleResult(FightResultEnum.CAUGHT);
+
+          try {
+            const userId = "02fea148-e9dc-4cae-89aa-8db50df0dd48"; // Replace with actual user ID
```

Comment:
why its hardcoded?

Reply:
About the hardcoded ID, it was to prepare for authentication. After merging, I will need to integrate the user with the service that fetches the data using the ID of the logged-in user.